java/install.sh

```diff
@@ -1,45 +1,33 @@
 #!/bin/bash -e
 
-echo "================= Installing openjdk-10-jdk ==================="
+echo "================= Installing openjdk-11-jdk ==================="
 mkdir -p /usr/lib/jvm && cd /usr/lib/jvm
-wget "https://download.java.net/java/GA/jdk10/10.0.2/19aef61b38124481863b1413dce1855f/13/openjdk-10.0.2_linux-x64_bin.tar.gz"
-tar -xzf openjdk-10.0.2_linux-x64_bin.tar.gz
-mv jdk-10.0.2/ java-10-openjdk-amd64
+wget "https://download.java.net/java/ga/jdk11/openjdk-11_linux-x64_bin.tar.gz"
```

Comment:
The version number should come from an env var.

Reply:
ok